client/components/date-range/use-shortcuts.ts

```diff
@@ -1,6 +1,6 @@
 import config from '@automattic/calypso-config';
 import { createSelector } from '@automattic/state-utils';
-import { translate, useTranslate } from 'i18n-calypso';
+import { translate as i18nCalypsoTranslate, useTranslate } from 'i18n-calypso';
```

Comment:
Is this also an issue (script not following import aliases)?

Reply:
No, as long as the translatable string is wrapped in a `translate` call it works.